cardano-shell.cabal

```diff
@@ -51,6 +52,8 @@ library
   default-language:    Haskell2010
   default-extensions:  NoImplicitPrelude
                        OverloadedStrings
+                       DeriveGeneric
```

Comment:
I don't really think we need these in default-extensions.
People should have visibility when looking at the source code and adding this here removes some of that visibility.

Reply:
Removed.